packages/web-components/src/components/ic-top-navigation/ic-top-navigation.tsx

```diff
@@ -330,17 +331,24 @@ export class TopNavigation {
                   {(hasTitle || isSlotUsed(this.el, "app-title")) && (
                     <Component class="title-link" {...attrs}>
                       {this.hasAppIcon && (
-                        <div class="app-icon-container" aria-hidden="true">
+                        <div class="app-icon-container">
```

Comment:
just checking the reason for removing the aria-hidden. previously I think this was added as the icon does add any extra information. happy if it's been agreed though!

Reply:
the icon used to be aria-hidden because we assumed only an image would be put in there. (since the slot is 'app-icon') but we didn't tell developers this was the case and it tripped up one of our customer applications

Reply:
Yeah we need to remove the aria-hidden now so that when people slot links in then they'll get read out. Like @GCHQ-Developer-299 mentioned, it tripped up a customer recently